handlers/validator.go

```diff
@@ -1127,13 +1130,26 @@ func ValidatorProposedBlocks(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, "Internal server error", http.StatusInternalServerError)
 			return
 		}
+
+		epochs := make([]uint64, 0, len(epochCount))
+		for key := range epochCount {
+			epochs = append(epochs, key)
+		}
+
+		incomeData, err = db.BigtableClient.GetValidatorIncomeDetailsIndexedMultiple([]uint64{index}, epochs)
```

Comment:
Why are CL Rewards only shown if EL rewards are present? CL proposer reward for slots exist regardless of whether the chain already has an EL, right?

Reply:
Oh, those did not work as I expected them to.
I fixed this so now there are 3 possibilities for the CL reward:

- Pending if the epoch is not yet finalized
Also a minute for the latest finalized epoch since the data is not immediately in the bigtable
- The ETH value from the bigtable
0 ETH is possible
- "-" if it is an old epoch and there is no data in the bigtable

I used validator 15500 on prater to check all of these.